src/Microsoft.Health.Fhir.Shared.Tests/Samples.cs

```diff
@@ -22,6 +23,14 @@ public class Samples
 
         public const string SampleConvertDataResponse = "{ \"resourceType\": \"Bundle\" }";
 
+        /// <summary>
+        /// This returns invalid resource bytes from the json files
+        /// </summary>
+        public static byte[] GetInvalidResourceBytes()
```

Comment:
Since this seems very specific, I'd recommend using the existing function `Samples.GetJsonSample("InvalidObservation")`, then use `Encoding.UTF8.GetBytes()` in your test method

Reply:
Thanks, I moved Encoding.UTF8.GetBytes() in the test method. And here, I use GetJson("InvalidObservation") instead of GetJsonSample("InvalidObservation") since GetJsonSample will parse json into resource.